packages/store/src/types.js

```diff
@@ -67,35 +103,43 @@
  * @typedef {[number, number]} HydrateKey
  * @typedef {true} HydrateInit
  * @typedef {Object} HydrateHook
+ *
  * @property {(value: any) => HydrateKey} getKey
  * @property {(key: HydrateKey) => any} load
  * @property {(storeId: number) => void} drop
  */
 
 /**
- * An external store that decouples the closure data from the returned
- * "representative" instance.
- *
  * @template {Array<any>} A
  * @template {ExternalInstance} T
  * @callback MakeHydrateExternalStore
+ * An external store that decouples the closure data from the returned
+ * "representative" instance.
+ *
```

Comment:
Since I can't find any uses of External Stores, can we delete them?

Reply:
@FUDCo are these your's? Ok if I delete them?

Reply:
I have no idea what these are.  They're not mine.

Reply:
@michaelfig It looks like they're yours. Ok to delete?

Reply:
Verified with @michaelfig I could delete it. Gone. Thanks for suggesting! Resolving.